week-2/L-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,11 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
+  str = str.split("");
+  str[0] = str[0].toUpperCase();
+  return str.join("")
```

Comment:
Missed semi-colon :) And remember to not reassign the function parameter (`str`) in this case - just create a new variable